vuepress/docs/next/tutorials/create/mfe/epc.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebarDepth: 2
+---
+
+# Create an Entando Platform Capability 
+
+An Entando Platform Capability, or EPC, is a packaged capability in the form of a bundle that adds functionality to the platform such as additional UX controls in the App Builder. With an EPC, adding menu options, an API management page, or integrating a WCMS like Strapi from the App Builder is simplified. This tutorial demonstrates how to build an EPC from a simple React micro frontend bundle.
+
+## Prerequisites
+* A working instance of Entando
+* An existing [React MFE](react.md)
+
+
+## Convert the React Bundle into an EPC
+Working with the [React MFE Tutorial](react.md), the following steps convert the React bundle into an EPC by modifying the bundle descriptor and customizing the public path to serve static assets. 
+
+### Configure the Bundle discriptor
+Edit the bundle descriptor `entando.json` in the root bundle directory. Remove the `type` and `titles` attributes and add the code snippet below, replacing the `paths`, `label` and `url` with your values.
+
+``` json
+"type": "app-builder",
+"slot": "content",
+"paths": ["/YOUR-MFE-NAME"],
+"nav": [
+    {
+        "label": {
+        "en": "YOUR EPC LABEL IN ENGLISH",
+        "it": "YOUR EPC LABEL IN ITALIAN"
+        },
+        "target": "internal", 
+        "url": "/YOUR-MFE-NAME"
+    }
+],    
+```
+* `type`: Use `app-builder` for an EPC
+* `slot`: Placement of the EPC on a page
+* `paths`, `url`: The address where the user will land. An external URL can be entered. Otherwise, this string will become part of the URL in the App Builder. 
+* `label`: What appears in the App Builder menu
```

Comment:
I would have a group of fields for the Nav item. Or point to the CLI docs on this. That might  be better.
-> "The visible label for the navigation item in the App Builder Menu"